pyproject.toml

```diff
@@ -26,6 +34,7 @@ post_test = "python scripts/json_coverage.py"
 lint = "flake8 --doctests"
 "lint:fix" =  "yapf -r -i logdna scripts tests"
 "post_lint:fix" = "task lint"
+release = "semantic-release publish"
```

Comment:
is this used/useful now that this is driven by the makefile? or rather, should the makefile instead run these tasks vs calling `semantic-release` directly?

Reply:
update makefile to run this.